app/styles/_forms.less

```diff
@@ -86,3 +86,16 @@
   // the sections just above.
   margin-top: @line-height-computed;
 }
+
+// Wrap for inline radio and/or checkbox groups that are displayed inline.
+.radio-checkbox-group-inline {
+  display: block;
+  @media (max-width: @screen-xs-max) {
+    // switch default bootstrap margin to right side to enable left alignment of inputs at mobile
+    .radio-inline,
+    .checkbox-inline {
+      margin-left: 0;
+      margin-right: 10px;
+    }
```

Comment:
Any reason not to do this always (instead of max-width xs)? It seems like it could still be a problem with long labels at any width.

Would this work without needing the parent div `radio-checkbox-group-inline`?

```css
.radio-inline,
.checkbox-inline {
  margin-left: 0;
  margin-right: 10px;
}
```